src/core/components/image-detail/image-detail.directive.html

```diff
@@ -1,6 +1,6 @@
 <tr ng-class="::{muted: !images.length}">
     <td>
-        <span class="row-label">Afbeeldingen</span>
+        <span class="row-label">{{translateImageDetail('label')}}</span>
     </td>
     <td ng-if="::images.length">
         <ul class="list-unstyled media-list">
```

Comment:
Alt tags kunnen ook vertaald worden, en dynamische waarden kunnen geinterpoleerd worden. https://angular-translate.github.io/docs/#/guide/06_variable-replacement

Reply:
translation voor alt tag is ook toegevoegd